content/en/install.md

```diff
@@ -3,101 +3,206 @@ title: Installation
 sidebar: false
 ---
 
-Installations methods include:
+The recommended method of installing SciPy depends on your preferred workflow.
+The common workflows can roughly be broken down into the following
+categories:
 
-- [Distributions](#distributions)
-- [pip](#pip-install)
-- [conda](#pip-install)
-- [Package Manager](#package_manager)
-- [Source](#source)
+- [Project-based (e.g. `uv`, `pixi`)](#project-based) (recommended for new users)
+- [Environment-based (e.g. `pip`, `conda`)](#environment-based) (the traditional workflow)
+- [System package managers](#system-package-managers) (not recommended)
+- [Building from source](#building-from-source) (for debugging and development)
 
-Methods differ in ease of use, coverage, maintenance of old versions,
-system-wide versus local environment use, and control. With pip or
-Anaconda\'s conda, you can control the package versions for a specific
-project to prevent conflicts. Conda also controls non-Python packages,
-like MKL or HDF5. System package managers, like `apt-get`, install
-across the entire computer, often have older versions, and don\'t have
-as many available versions. Source compilation is much more difficult
-but is necessary for debugging and development. If you don\'t know which
-installation method you need or prefer, we recommend the Scientific
-Python Distribution [Anaconda](https://www.anaconda.com/download/).
+To install SciPy with [static type stubs],
+see [Installing with type stubs](#type-stubs).
 
-<a name="distributions"></a>
+[static type stubs]: https://typing.readthedocs.io/en/latest/guides/libraries.html
 
-# Scientific Python Distributions (recommended)
+<a name="project-based"></a>
 
-Python distributions provide the language itself, along with the most
-commonly used packages and tools. These downloadable files require
-little configuration, work on almost all setups, and provide all the
-commonly used scientific Python tools.
+## Project-based workflows
 
-[Anaconda](https://www.anaconda.com/download/) works on Windows, Mac,
-and Linux, provides over 1,500 Python packages, and is used by over 15
-million people. Anaconda is best suited to beginning users; it provides
-a large collection of libraries all in one.
+In project-based workflows, a project is a directory containing a manifest
```

Comment:
This is, according to the new text above, the recommended route for newcomers. So, the first thing a newcomer reads is "manifest", "lock-file", "dependencies". Yikes!

How about these more common use-cases:

- I found a snippet on StackOverflow and want to play with it in IPython or Jupyter.
- I am reading the Scientific Python lecture notes and want to try some examples.
- I am studying the SciPy API guide / other docs and want to try some code snippets I see.

When you make project-based the default installation path, the instructions that come with tutorials won't work. Nothing will work, as stated below, without quite a bit of future reading.

So I'll reiterate my earlier suggestion that you make this as simple as possible for someone who just wants to get going to see how the project works. And my strong feeling is that that route is pip or conda, and not starting a new project.

Reply:
> So, the first thing a newcomer reads is "manifest", "lock-file", "dependencies". Yikes!

Agreed 😅 I think this is solvable by moving this below to the discussion of how environment-based workflows are different. It made more sense before I swapped the sections.

> When you make project-based the default installation path, the instructions that come with tutorials won't work. Nothing will work, as stated below, without quite a bit of further reading.

For my understanding, could you point to some tutorial instructions for which this would be problematic?